app/models/concerns/hostext/operating_system.rb

```diff
@@ -0,0 +1,55 @@
+module Hostext
+  module OperatingSystem
+    extend ActiveSupport::Concern
```

Comment:
Could you please rather extract it to a separate class, e.g. HostTemplatePicker that would require a host and only return templates? Hiding it in concern that is included does not make the object any less fat and make testing harder. With this pattern, one could inject testing host object in unit tests for picker class.